pkg/controller/directvolumemigration/task.go

```diff
@@ -174,6 +179,18 @@ func (t *Task) Run() error {
 		if err = t.next(); err != nil {
 			return liberr.Wrap(err)
 		}
+	case CleanStaleRsyncResources:
+		// TODO Need to add some labels during DVM run to differentiate
```

Comment:
note: with #808 we made a call to try the DVM migration only once and in the future allowing a user-configurable policy on the number of retries. We should probably have the notion which retries number the certs belong to in the label selectors in order to avoid conflicts. but +1 on the TODO